charts/elasti/templates/elastiservice-crd.yaml

```diff
@@ -78,9 +78,16 @@ spec:
                     enum:
                       - deployments
                       - rollouts
+                      - Deployment
+                      - StatefulSet
+                      - Rollout
                     type: string
                   name:
                     type: string
+                required:
+                  - apiVersion
+                  - kind
+                  - name
```

Comment:
These required validation are usually considered breaking changes, however, in this case it is acceptable. 
This is because these fields are part of the core functionality, and are definitely required.